source/server-discovery-and-monitoring/tests/integration/minPoolSize-error.yml

```diff
@@ -0,0 +1,79 @@
+# Test SDAM error handling.
+runOn:
+    # failCommand appName requirements
+    - minServerVersion: "4.4"
```

Comment:
This test only works correctly on 4.9+ because of [SERVER-49336](https://jira.mongodb.org/browse/SERVER-49336).

Reply:
fixed